components/object-property-list/object-property-list-item.js

```diff
@@ -44,14 +45,21 @@ export class ObjectPropertyListItem extends RtlMixin(LitElement) {
 			:host([dir="rtl"]) .item-icon {
 				margin: -0.1rem 0 0 0.3rem;
 			}
+			.d2l-skeletize {
+				display: inline-block;
+				width: 100px;
+				height: 0.7rem;
+			}
 		`];
 	}
 
 	render() {
-		return html`
+		return !this.skeleton ? html`
 			${this._renderIcon()}
 			<span>${this.text}</span>
 			${this._renderSeparator()}
+		` : html`
+			<div class="d2l-skeletize"></div>
```

Comment:
I am creating a separate div so that we always show a skeleton block no matter the length of the text.

I am doing regardless of the text length because while loading we might not have the text available yet

Reply:
I have a look at the `d2l-skeletize-<width>` helper classes -- they're intended for exactly this purpose.

Reply:
Also, do we maybe want the icon and the separator to show up when it's in skeleton mode? If so, you might be able to just apply `d2l-skeletize-10` (or whatever size looks good) to the text `<span>` directly when in skeleton-mode.

Reply:
It seems to me that we would still want the separators (and maybe icons?) rendered when the item is skeletized. Also from the docs it looks like `d2l-skeletize-<width>` is for block elements - the items are `inline` by default. Would it make more sense for consumers to do something like:

`<d2l-object-property-list-item text="Placeholder" skeleton>`

And then the skeleton will just be the width of the placeholder? Not sure what's best for accessibility here, whether we'd want a descriptive placeholder or for it to read as nothing...

Reply:
So according to [this](http://proddevutil.desire2learn.d2l/d2c/content/product%20design/design.d2l%20archive/Skeleton%20Page%20Loaders%20%E2%80%93%20Daylight%20Design%20System.html) we do not want to display the icons.

Also, I am not sure if I am not doing it right, but `d2l-skeletize-<width>` did not help me. It seems to set the height and width to 0 if there is nothing inside the `div`.